dotcom-rendering/src/web/components/Callout/FieldLabel.tsx

```diff
@@ -16,15 +20,22 @@ const optionalTextStyles = css`
 	padding-left: 5px;
 `;
 
-export const FieldLabel = ({ formField }: { formField: CampaignFieldType }) => (
-	<label css={fieldLabelStyles} htmlFor={formField.name}>
+type Props = {
+	formField: CampaignFieldType;
+	hideLabel?: boolean;
+};
+
+export const FieldLabel = ({ formField, hideLabel }: Props) => (
+	<label
+		css={[fieldLabelStyles, hideLabel && visuallyHidden]}
+		htmlFor={formField.name}
+		hidden={formField.hideLabel}
```

Comment:
The `hideLabel` property is now used directly on the label property. If this is true, we implement the `visuallyHidden` style from Source. I believe this now should meet accessibility requirements because I am essentially copying the same behaviour as Source (see here: https://github.com/guardian/csnx/blob/main/libs/%40guardian/source-react-components/src/label/Label.tsx)

Reply:
nice write up, tom!